qt/python/mantidqt/plotting/functions.py

```diff
@@ -227,10 +229,31 @@ def plot(workspaces, spectrum_nums=None, wksp_indices=None, errors=False,
             else:
                 ax.axis('off')
     else:
-        ax = overplot if isinstance(overplot, MantidAxes) else axes[0]
+        if isinstance(overplot, MantidAxes):
+            ax = overplot
+            x, y = 0, 0
```

Comment:
x and y should be initialized outside this loop as you have used them later so there is a chance they may not be initialized i.e. for tiled plots